tool/gravity/cli/config.go

```diff
@@ -506,7 +506,9 @@ func (i *InstallConfig) RunLocalChecks() error {
 func (i *InstallConfig) BootstrapSELinux(ctx context.Context, printer utils.Printer) error {
 	logger := log.WithField(trace.Component, "selinux")
 	if !i.shouldBootstrapSELinux() {
-		logger.Info("SELinux disabled with configuration.")
+		if !i.FromService {
+			logger.Info("SELinux disabled with configuration.")
```

Comment:
This is to avoid confusing message from the service log when in fact SELinux is enabled.